raiwidgets/raiwidgets/model_analysis_dashboard.py

```diff
@@ -5,7 +5,7 @@
 
 from flask import jsonify, request
 
-from responsibleai import ModelAnalysis
+from responsibleai import RAIInsights
 
 from .dashboard import Dashboard
 from .model_analysis_dashboard_input import ModelAnalysisDashboardInput
```

Comment:
Is this unrelated to this change?

Reply:
yes that will be changed in follow-up PRs as well:
1.) change ModelAnalysisDashboard to ResponsibleAIDashboard
2.) change notebooks and any internal constants that are still left